app/views/languages/repository.erb

```diff
@@ -0,0 +1,67 @@
+<div class="container">
+  <section class="contributing-intro jumbotron">
+    <h1>Exercism Repositories</h1>
+
+    <p class="lead">
+      <strong>What are the repositories you can contribute to?</strong><br />
+    </p>
+
+    <p> Exercism is a project comprised of several repositories and these can be divided basically into:</p>
+
+     <h2> Basic components </h2>
+
+     <ul>
+      <li><strong><a href="https://github.com/exercism/exercism.io">The website</a></strong>: Holds the code that people submit. Browse and review solutions, and have conversations.</li>
+      <li><strong><a href="https://github.com/exercism/cli">Command line interface</a></strong>: Used to fetch exercises and submit solutions.</li>
+      <li><strong><a href="https://github.com/exercism/x-api">Problems API</a></strong>: Delivers exercises. Consumed by the Exercism command-line client, as well as the Exercism website.</li>
```

Comment:
I'd feel better about this section if we could have a consistent tense/format for the three bullet pointns. "Does X. Does Y. Does Z."